e107_languages/English/English.php

```diff
@@ -139,5 +139,5 @@
 define("LAN_REQUIRED_BLANK", "Required field(s) were left blank.");
 define("LAN_PLEASEWAIT", "Please Wait");
 define("LAN_CHOOSE_FILE", "Choose a file");
-
+define("LAN_NEWS", "News");
```

Comment:
Should be in e107_plugins/news/languages/English.php and in the correct format (using numbers). and then loaded in e_search.php